src/components/menu/js/menuServiceProvider.js

```diff
@@ -476,7 +480,7 @@ function MenuProvider($$interimElementProvider) {
         top: Math.round(position.top),
         left: Math.round(position.left),
         // Animate a scale out if we aren't just repositioning
-        transform: !opts.alreadyOpen ? $mdUtil.supplant('scale({0},{1})', [scaleX, scaleY]) : undefined,
+        transform: !opts.alreadyOpen && !opts.isAnimating ? $mdUtil.supplant('scale({0},{1})', [scaleX, scaleY]) : undefined,
```

Comment:
Yes, the actual problem is, that on IOS the minimal-ui will trigger a orientantionchange / resize event after the menu animation is started. 

I already thought about adding this up to the actual $$rAF but it would make more sense to let it update the position except the `scale` from `undefined` to `scale x/y`